ccpp/data/GFS_typedefs.F90

```diff
@@ -2896,6 +2903,18 @@ subroutine coupling_create (Coupling, IM, Model)
 !     Coupling%sfc_alb_vis_dir_cpl   = clear_val
 !     Coupling%sfc_alb_vis_dif_cpl   = clear_val
 
+      allocate (Coupling%dusfcino_cpl      (IM))
```

Comment:
Should these be allocated only if `use_med_flux` is `.true.`? That is at least what the `active` attribute in `GFS_typedefs.meta` is saying.

Reply:
@climbfuji yes. it was my initial plan but when I protect it with the if statement I was getting error in the initial implementation. Anyway, I'll look at this part more carefully in try to make it work. I think I could fix it easily.

Reply:
@climbfuji okay. i fixed the issue with https://github.com/NOAA-EMC/fv3atm/pull/517/commits/83496d2f8c9fb1d327836208f0baa78931438e3c